### ISSUE TITLE
San Francisco Marina District Loop 

### ISSUE THREAD

**Issue by XiCynx** [NecronomiconCoding/NecroBot#807]
## Loop Running
**Marina District(near Golden Gate Bridge), San Francisco, CA**
Screenshot: <a href="http://www.directupload.net/file/d/4429/lmfefwnm_png.htm">Image</a>
Delimiter = Period: <a href="http://www.filedropper.com/marinadistrictggloop">MarinaDistrict(GG)_Loop.gpx</a>
Starting Location: `lat="37.8081395285097", lon="-122.42664098739624"`

Nests: Squirtle, Bulbasaur, Magikarp
Screenshot: <a href="https://i.imgur.com/fQtUmxq.jpg">Image</a>

Notes: Best to keep Move Speed <100 so it grabs all the Pokestops.

Edit: If there is a better way to add items to the Wiki, let me know.  I plan on making a lot of these routes.  If I am able to Edit the Wiki, that would probably be best.  Thanks!

**Comment by spot88** [comment]
Mod place this here: 
https://github.com/NecronomiconCoding/NecroBot/wiki/Locations-&-GPX-Route-Files

-- Close this.

**Comment by spot88** [comment]
@XiCynx Could you add one with comma aswell?

**Comment by skupfer** [comment]
Take this complete version of the wiki. I added several more and as always I added both period and comma delimited versions.

@XiCynx as you can see under General Information, I politely asked to provide both. If you really want to take care for this wiki, please do so, too. We have a lot of users in the location channel with problems related to the delimiter :)

http://pastebin.com/qEK9tD0B

**Comment by XiCynx** [comment]
Sorry about not having a comma comma delimited version as well.  I can definitely create those in the future without issue.  @skupfer thanks for the complete version for the wiki format.  I did have a question though, how would I go about editing it on the main page?  I went over there and didn't see anything that would allow me to actually make the changes necessary to update what you have in the pastebin.  Or do I need to request permission from someone higher up the ladder?
